servatrice/servatrice.sql

```diff
@@ -28,6 +28,7 @@ CREATE TABLE IF NOT EXISTS `cockatrice_users` (
   `admin` tinyint(1) NOT NULL,
   `name` varchar(35) NOT NULL,
   `realname` varchar(255) NOT NULL,
+  -- this field is deprecated and will not be used, this could be removed in a database migration
   `gender` char(1) NOT NULL,
```

Comment:
we'll want to remove it here, bump db version, and create migrate script in this pr

Reply:
I'm not sure how to do a db migration, it's some sql script?

Reply:
Correct, we have a folder of them within the repo already